deployment/modules/gcp/storage/main.tf

```diff
@@ -65,3 +65,47 @@ resource "google_spanner_database" "dedup_db" {
     "CREATE TABLE IDSeq (id INT64 NOT NULL, h BYTES(MAX) NOT NULL, idx INT64 NOT NULL,) PRIMARY KEY (id, h)",
   ]
 }
+
```

Comment:
I'm wondering whether it would make sense to move this to a different module? Thoughts? It would be good to make it explicit that this is decoupled from the storage bits, and that this has not been blessed for production.

Reply:
That's a good idea. I looked into the module design. It requires a root module when we separate the storage module and key module.

Let's do that in the next pull request.

Reply:
SGTM, and wen can also add in the README that this is really, for tests, an not production use.